web/themes/custom/digital_gov/css/new/_content.scss

```diff
@@ -14,7 +14,7 @@
   // Select all the links with href that start with "http" and do not contain ".gov"
   // See https://css-tricks.com/attribute-selectors/
   a[href^="http"]:not([href*=".gov"])::after {
-    background: url("../uswds/img/external-link.svg") no-repeat 0 0;
+    background: url("#{$theme-image-path}/external-link.svg") no-repeat 0 0;
```

Comment:
Definition of this var in `_uswds_theme.css` is this. Which is making me wonder if that's a relative path still ... how does it fix things?

`$theme-image-path: "../uswds/img",`